compiler-cli/src/remove.rs

```diff
@@ -0,0 +1,42 @@
+use std::path::{Path, PathBuf};
+
+use gleam_core::{
+    error::{FileIoAction, FileKind},
+    Error, Result,
+};
+
+use crate::{cli, fs, UseManifest};
+
+pub fn command(packages: Vec<String>) -> Result<()> {
+    // Read gleam.toml so we can remove deps from it
+    let mut toml = fs::read("gleam.toml")?
+        .parse::<toml_edit::Document>()
+        .map_err(|e| Error::FileIo {
+            kind: FileKind::File,
+            action: FileIoAction::Parse,
+            path: PathBuf::from("gleam.toml"),
+            err: Some(e.to_string()),
+        })?;
+
+    // Insert the new deps
+    for package_to_remove in packages.iter() {
+        #[allow(clippy::indexing_slicing)]
+        let _ = toml["dependencies"]
+            .as_table_mut()
+            .and_then(|deps| deps.remove(&package_to_remove));
+        #[allow(clippy::indexing_slicing)]
+        let _ = toml["dev-dependencies"]
+            .as_table_mut()
+            .and_then(|deps| deps.remove(&package_to_remove));
+    }
+
+    // Write the updated config
+    fs::write(Path::new("gleam.toml"), &toml.to_string())?;
+    let paths = crate::project_paths_at_current_directory();
+    _ = crate::dependencies::download(&paths, cli::Reporter::new(), None, UseManifest::No)?;
```

Comment:
Not using the manifest will change the versions of all the other deps, let's use it so only the packages specified change.

Reply:
@lpil done :)